### ISSUE TITLE
Windows IE cannot copy and paste images

### ISSUE THREAD

**Issue by yunyongEthan** [JiHong88/SunEditor#668]
My version was 2.33.0, but it was solved when I uploaded it to 2.37.0.
Thanks.

**Comment by yunyongEthan** [comment]
Oh, i do

**Comment by yunyongEthan** [comment]
It works if you copy the entire interior image area. 
But then there was another issue.
I tried to paste the image that was copied elsewhere, but it is not working properly in window IE (ver.11).

Thanks.

**Comment by JiHong88** [comment]
Does this mean that you can't paste an image copied from elsewhere?

**Comment by yunyongEthan** [comment]
Then, is it a situation that cannot be solved due to the IE issue?

**Comment by JiHong88** [comment]
Yup, `clipboardData.setData('text/html', HTML);` is not working in IE.

https://github.com/JiHong88/SunEditor/blob/master/src/lib/core.js#L6864
https://github.com/JiHong88/SunEditor/blob/master/src/lib/core.js#L6853
